README.md

```diff
@@ -59,61 +59,71 @@ recommended method of start using PipelineWise.
 
 1. Build an executable docker images that has every required dependency and it's isolated from your host system:
 
-```sh
-$ docker build -t pipelinewise:latest .
-```
+    ```sh
+    $ docker build -t pipelinewise:latest .
+    ```
 
 2. Once the image is ready, create an alias to the docker wrapper script:
 
-```sh
-$ alias pipelinewise="$(PWD)/bin/pipelinewise-docker"
-```
+    ```sh
+    $ alias pipelinewise="$(PWD)/bin/pipelinewise-docker"
+    ```
 
 3. Check if the installation was successfully by running the `pipelinewise status` command:
 
-```sh
-$ pipelinewise status
-
-Tap ID    Tap Type      Target ID     Target Type      Enabled    Status    Last Sync    Last Sync Result
---------  ------------  ------------  ---------------  ---------  --------  -----------  ------------------
-0 pipeline(s)
-```
+    ```sh
+    $ pipelinewise status
+    
+    Tap ID    Tap Type      Target ID     Target Type      Enabled    Status    Last Sync    Last Sync Result
+    --------  ------------  ------------  ---------------  ---------  --------  -----------  ------------------
+    0 pipeline(s)
+    ```
 
 You can run any pipelinewise command at this point. Tutorials to create and run pipelines is at https://transferwise.github.io/pipelinewise/installation_guide/creating_pipelines.html .
 
+**PS**: 
+
+For the tests to work, run the commands inside a PipelineWise container.
+To create, start and get a bash shell in the container:
+
+```sh
+$ docker-compose up -d pipelinewise
+$ docker exec -it pipelinewise_dev bash
+```
+
```

Comment:
Maybe you can add a link as well to the page with the details of developing and running tests via docker-compose: https://github.com/transferwise/pipelinewise/blob/master/dev-project/README.md

Reply:
Hey, I'm glad I made my first contribution.

There is already a link to that page in the [Developing with Docker](https://github.com/transferwise/pipelinewise/blob/master/README.md#developing-with-docker) section.
Please let me know in case I misunderstood your comment.

Reply:
ah yes, sorry you're right. Ignore this comment.
PR approved, OK to merge.